pkg/xds/envoy/routes/default_route_configurer_test.go

```diff
@@ -70,6 +70,7 @@ var _ = Describe("DefaultRouteConfigurer", func() {
                           version: v2
                     name: backend
                     weight: 70
+                  totalWeight: 100
```

Comment:
Can you add a test that does not add to 100, so we can continuously verify this piece of functionality is OK?